graylog2-server/src/main/java/org/graylog2/plugin/configuration/ConfigurationRequest.java

```diff
@@ -150,6 +152,8 @@ public Configuration filter(Configuration config) {
                     }
                     break;
                 case TextField.FIELD_TYPE:
+                case ListField.FIELD_TYPE:
+                    break;
```

Comment:
This will break the `TextField.LIST_TYPE` case, because it now also returns on the `break`. We need to implement a separate case for `ListField.FIELD_TYPE` that also sets the list value like the `DropdownField.FIELD_TYPE` does. (just for lists) This probably needs a `listIsSet(String)` method implementation in `org.graylog2.plugin.configuration.Configuration`.

Reply:
oh okay. will add method.